exchanges/binance/binance_wrapper.go

```diff
@@ -190,8 +191,20 @@ func (b *Binance) ModifyOrder(orderID int64, action exchange.ModifyOrder) (int64
 }
 
 // CancelOrder cancels an order by its corresponding ID number
-func (b *Binance) CancelOrder(orderID int64) error {
-	return common.ErrNotYetImplemented
+func (b *Binance) CancelOrder(order exchange.OrderCancellation) (bool, error) {
+	orderIDInt, err := strconv.ParseInt(order.OrderID, 10, 64)
+
+	if err != nil {
+		return false, err
+	}
+
+	_, err = b.CancelExistingOrder(order.CurrencyPair.Pair().String(), orderIDInt, order.AccountID)
```

Comment:
You can use exchange.FormatExchangeCurrency(b.Name, p).String(), this will ensure that the pair is formatted as per the exchanges expected format.